transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java

```diff
@@ -828,4 +847,49 @@ Entry recycleAndGetNext() {
             return next;
         }
     }
+
+    // the cumulator copied from ByteToMessageDecoder
```

Comment:
@Scottmitch hm it's not worth it because taking it away from BTMDecoder would be api breaking. So there would need to be two copies anyway.